source/calibration.rst

```diff
@@ -161,7 +161,7 @@ Torso Calibration Procedure
 Follow these steps in order to verify or calibrate the torso position
 sensors:
 
-1. Move torso to lowest position, and tuck the arm.
+1. Move torso to lowest position, and tuck the arm : ``rosrun fetch_teleop tuck_arm.py``
```

Comment:
Hi @NWChen, running that command will invalidate the torso calibration, as it does not put the torso at the lowest position.

Instead, perhaps the below modification would be an acceptable tweak?

```suggestion
1. Move torso to its lowest position, and manually tuck the arm. (Note: do not use the tuck-arm
   feature of the tele-op controller, as it moves the torso higher than its lowest position).
```

Additionally, note that the tuck_arm.py script is what is invoked by button 6 (see [here](http://docs.fetchrobotics.com/teleop.html) ).  Since keyboard teleop doesn't give a way to lower the torso, we generally expect users will use their PS3 controller to lower the torso.  Instructing the user to use both the PS3 controller and the keyboard teleop at the same time would be a bit confusing, too, I think.